src/viewport/admin/__init__.py

```
@@ -1,6 +1,7 @@
 """Admin panel module for Viewport application."""
 
+from viewport.admin.application import ViewportAdmin
 from viewport.admin.auth import AdminAuth
 from viewport.admin.views import GalleryAdmin, PhotoAdmin, ProjectAdmin, ShareLinkAdmin, UserAdmin
 
-__all__ = ["AdminAuth", "UserAdmin", "ProjectAdmin", "GalleryAdmin", "PhotoAdmin", "ShareLinkAdmin"]
+__all__ = ["AdminAuth", "ViewportAdmin", "UserAdmin", "ProjectAdmin", "GalleryAdmin", "PhotoAdmin", "ShareLinkAdmin"]
```

Ruff (0.16.1)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)